README.md

```diff
@@ -308,6 +308,43 @@ This modality provides three options:
 | V_1_0_standard            | 1   |
 | V_1_0_custom              | 2   |
 
+#### EventsMode
+
+The `EventsMode` modality determines how the installed instance of CEP-78 will handle the recording of events that occur from interacting with the contract.
+
+The modality provides two options:
+
+1. `NoEvents`: This modality will signal the contract to not record events at all.
+2. `CEP47`: This modality will signal the contract to record events in a manner similar to that used for CEP47. Events are stored as a `BTreeMap` within dictionary in the contract's context. The events can be retrieved directly via their dictionary entry using the JSON-RPC, with more information on this process available [here](https://docs.casperlabs.io/dapp-dev-guide/writing-contracts/dictionaries/).
+
+Events are recorded as one of the following `event_type`s alongside the listed information.
+
+* `Mint`
+   - The `Key` of the recipient.
+   - The [`TokenIdentifier`](#nftidentifiermode) for the `token_id`.
+
+* `Burn`
+   - The `Key` of the owner.
+   - The [`TokenIdentifier`](#nftidentifiermode) for the `token_id`.
+
+* `Approve`
```

Comment:
Hi,

I think there is an ApproveAll too, variable set and migration 

You can see the current full list in the opened PR 

https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/150/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R676